src/api/Models/Borehole.cs

```diff
@@ -334,15 +334,15 @@ public class Borehole : IChangeTracking, IIdentifyable
     public double? ReferenceElevation { get; set; }
 
     /// <summary>
-    /// Gets or sets the <see cref="Borehole"/>'s Qt reference elevation id.
+    /// Gets or sets the <see cref="Borehole"/>'s reference elevation precision id.
     /// </summary>
     [Column("qt_reference_elevation_id_cli")]
-    public int? QtReferenceElevationId { get; set; }
+    public int? ReferenceElevationPrecisionId { get; set; }
 
     /// <summary>
-    /// Gets or sets the <see cref="Borehole"/>'s Qt reference elevation.
+    /// Gets or sets the <see cref="Borehole"/>'s reference elevation precision.
     /// </summary>
-    public Codelist? QtReferenceElevation { get; set; }
+    public Codelist? ReferenceElevationPrecision { get; set; }
 
     /// <summary>
     /// Gets or sets the <see cref="Borehole"/>'s reference elevation type id.
```

Comment:
Q: War das Umbenennen schon länger geplant? Bzw. wurde die Gelegenheit einfach zu diesem Zeitpunkt genutzt?

Reply:
Habe mit Michal abgemacht, dass wir im CSV File Export die Namen aus unserem Model als Header verwenden können, statt auf die aktuellen Header wie sie im CSV-Import sind zu mappen. (So können wir uns das mapping dann auch beim Import sparen). Bei der Gelegenheit ist aufgefallen, dass die Namen nicht (mehr) den im UI und in den Domains verwendeten entsprechen, also habe ich die angepasst.

Reply:
Okay, super